perf/rollup_factory.go

```diff
@@ -145,7 +145,7 @@ type documentThroughput struct{}
 
 const (
 	documentThroughputName    = "DocumentThroughput"
-	documentThroughputVersion = 0
+	documentThroughputVersion = 2
```

Comment:
Nit: can't this just changed to `1`?

Reply:
TBH, I would have been fine to reject the review on this. Will fix.

Reply:
Also good catch.